01-HelloPrism/src/PrismSample.Android/SplashActivity.cs

```diff
@@ -11,8 +11,6 @@ namespace PrismSample.Droid
               NoHistory = true)]
     public class SplashActivity : AppCompatActivity
     {
-        static readonly string TAG = "X:" + typeof(SplashActivity).Name;
-
```

Comment:
This removes the property but would cause all of the projects to fail to build since the following OnCreate references it. Be sure to remove the OnCreate in the SplashActivity.

Reply:
I'll remove the oncreate from all the SplashActivity.cs files

Reply:
Removed  Log.Debug(TAG, "SplashActivity.OnCreate"); from OnCreate()
Or do you want me to remove the whole OnCreate method?

Reply:
Be sure to remove the OnCreate in the SplashActivity.